File: sdk/samples/WCFSamples/TechnologySamples/Basic/Binding/WS/FederationHttp/VB/SecurityTokenService/IWSTrust.vb
' Copyright (c) Microsoft Corporation.  All Rights Reserved.

Imports System
Imports System.ServiceModel
Imports System.ServiceModel.Channels

Namespace Microsoft.ServiceModel.Samples

    <ServiceContract()> _
    Interface IWSTrust

        <OperationContract(Action:=Constants.Trust.Actions.Issue, ReplyAction:=Constants.Trust.Actions.IssueReply)> _
        Function Issue(ByVal request As Message) As Message

    End Interface

End Namespace

File: sdk/samples/WCFSamples/TechnologySamples/Basic/Binding/WS/FederationHttp/CS/SecurityTokenService/IWSTrust.cs
//-----------------------------------------------------------------------------
// Copyright (c) Microsoft Corporation.  All rights reserved.
//-----------------------------------------------------------------------------
using System;

using System.ServiceModel;
using System.ServiceModel.Channels;

namespace Microsoft.ServiceModel.Samples
{
    [ServiceContract]
    interface IWSTrust
    {
        [OperationContract(Action = Constants.Trust.Actions.Issue, ReplyAction = Constants.Trust.Actions.IssueReply)]
        Message Issue(Message request);
    }
}
